toolkit/themes/shared/design-system/design-tokens.json

```diff
@@ -135,40 +101,42 @@
         "value": "#5a3100"
       }
     },
-    "white": "#ffffff"
-  },
-  "text": {
-    "color": {
-      "prefers-contrast": {
-        "value": "CanvasText"
-      },
-      "deemphasized": {
-        "prefers-contrast": {
-          "value": "inherit"
-        }
-      },
-      "brand": {
-        "light": {
-          "value": "{color.gray.100}"
-        },
-        "dark": {
-          "value": "{color.gray.05}"
-        },
-        "deemphasized": {
-          "light": {
-            "value": "color-mix(in srgb, currentColor 69%, transparent)"
-          },
-          "dark": {
-            "value": "color-mix(in srgb, currentColor 75%, transparent)"
-          }
-        }
+    "white": {
+      "value": "#ffffff"
+    },
+    "background": {
+      "critical": {
+        "value": "{color.red.05}",
+        "darkValue": "{color.red.80}"
+      },
+      "information": {
+        "value": "{color.blue.05}",
+        "darkValue": "{color.blue.80}"
+      },
+      "success": {
+        "value": "{color.green.05}",
+        "darkValue": "{color.yellow.80}"
+      },
+      "warning": {
+        "value": "{color.yellow.05}",
+        "darkValue": "{color.blue.80}"
       }
     }
   },
-  "platform": {
-    "value": "currentColor",
+  "text": {
+    "color": {
+      "value": "CanvasText"
+    },
+    "brand": {
+      "value": "{color.gray.100}",
+      "darkValue": "{color.gray.05}"
+    },
+    "platform": {
+      "value": "currentColor"
+    },
     "deemphasized": {
-      "value": "color-mix(in srgb, currentColor 60%, transparent)"
+      "value": "color-mix(in srgb, currentColor 60%, transparent)",
```

Comment:
We're going to need a way to write this token without using `color-mix` as that's probably not a thing on mobile platforms. Not something that needs to be solved right now, but good to think about.

Reply:
Maybe we could key this using the array values?

```json
"deemphasized": {
  "value": ["color-mix", "in srgb", "currentColor 60%", "transparent"]
}
```

I believe that would let us try to come up with a solution in a transformer for mobile, and we could potentially even have a generic transformer for light-dark, etc

```json
"critical": {
  "value": ["light-dark", "{color.red.05}", "{color.red.80}"]
}
```